src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -207,6 +212,23 @@ public long getFilteredCount() {
 		return new ResponseEntity<>(observationUnitTable, HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "It will retrieve all the observation units in a simple JSON array table format, this will be used as input data for Visualization tool.",
+		notes = "It will retrieve all data of factors and variate at observation/sub-observation level. Returns data as a simple JSON Array table format.")
+	@RequestMapping(value = "/{cropname}/studies/{studyId}/datasets/{datasetId}/observationUnits/visualization", method = RequestMethod.POST)
+	@ResponseBody
+	public ResponseEntity<List<Map<String, Object>>> getObservationUnitTableAsJSONArray(@PathVariable final String cropname, //
```

Comment:
Omit in comment:
> this will be used as input data for Visualization tool.

Reply:
It's important to include in API notes that only variables (let's not use factors and variates anymore) specified in `filterColumns` will be retrieved in the map

Reply:
**visualization** as part of the endpoint is unclear/improper. What does it mean? The service should be agnostic of where the data is being/or will be used. 

We can rename:
1. method to **getObservationUnitsAsMapList**
2. endpoint to /{cropname}/studies/{studyId}/datasets/{datasetId}/observationUnits/mapList